pytorch_lightning/accelerators/gpu.py

```diff
@@ -83,63 +82,3 @@ def teardown(self) -> None:
     def auto_device_count() -> int:
         """Get the devices when set to auto."""
         return torch.cuda.device_count()
-
-
-def _get_nvidia_gpu_stats(device: torch.device) -> Dict[str, float]:
```

Comment:
I'm not against making it public, but I would keep it in this file. the utilities file keeps growing and growing

and `from pytorch_lightning.accelerators.gpu import get_nvidia_gpu_stats` seems more natural to me.

Reply:
Same here !

Reply:
fine with me. I just want it to be public

Reply:
updated!